### ISSUE TITLE
Spinner SVG fails WCAG 2.3.1

### ISSUE THREAD

**Issue by AlmeroSteyn** [tenon-io/tenon-ui#26]
As per the email from Mark Rogers:

> Hi Karl
> 
> Quick heads up – there might be a WCAG 2.3.1 ‘Three Flashes or Below
> Threshold’ fail in the Tenon React spinner GIF:
> 
> https://blog.tenon.io/tenons-accessible-react-components-coming-soon-to-an-npm-repo-near-you/
> 
> https://blog.tenon.io/wp-content/uploads/2018/09/Spinner.gif
> 
> Reasons I think it’s a fail of 2.3.1
> https://www.w3.org/TR/WCAG20/#seizure-does-not-violate:
> 
> Graphic is 717 x 717 pixels (above threshold size in
> https://www.w3.org/TR/WCAG20/#general-thresholddef )
> Qualifies as ‘general flash’ because the navy to white change is more
> than a 10% change in relative luminance
> There are approx. 6 navy to white flashes per 1 second period (I
> counted 90 in 13 seconds)
> 
> Not sure exactly how this is rendered by the React components, but if
> it’s displayed above certain sizes it will trigger WCAG 2.3.1. I think
> it always violates 2.3.2 ‘Three Flashes at any size’. Also need to
> take browser zoom in account now, so even small flashing graphics can
> cause seizure problems when zoomed.
> 
> Best Regards
>
